sympy/geometry/line.py

```diff
@@ -640,26 +706,48 @@ def is_perpendicular(l1, l2):
         return S.Zero.equals(l1.direction.dot(l2.direction))
 
     def is_similar(self, other):
-        """
-        Return True if self and other are contained in the same line.
+        """Whether lines are same or different.
+
+        Parameters
+        ==========
+
+        other : GeometryEntity
+
+        Returns
+        =======
+
+        bool :
+            True : if self and other are contained in the same line.
+            False : otherwise.
```

Comment:
Don't think this is much useful to the user. 
These messages should be regarding the lines, not the implimentation details.

Reply:
I think then it's first line summary must be changed as that too will not of any use for user.